### PR TITLE
fixes extra space in parentDiv

### PR DESCRIPTION
**Pull request by anuragnegi000**
**What kind of change does this PR introduce?**

bugfix

**Issue Number:**

Fixes #
#2071 

**Did you add tests for your changes?**

No not required

**Snapshots/Videos:**

**If relevant, did you update the documentation?**
Before:-
beforep1

After:-
latest

**Summary**

This change is motivated by the need to optimize space utilization within the parent container, which was previously taking up extra space due to the inclusion of the ProfileDropdown component. By removing this excess space, aiming to achieve a cleaner and more visually appealing layout.

**Does this PR introduce a breaking change?**

No
**Other information**

--
**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

## Summary by CodeRabbit

- **New Features**
	- Enhanced layout and visibility of the profile dropdown component for improved user accessibility.
	- Improved styling consistency across various user portal screens by refining padding and background colors.

- **Bug Fixes**
	- Resolved layout issues by adjusting positioning and padding in multiple component styles.

- **Chores**
	- Updated CSS classes across several modules to streamline design and improve user interface aesthetics.

### CODE REVIEW

File: package-lock.json
Comment (pranshugupta54) [inline package-lock.json:1]: We shouldn't be having any changes to this file bcuz no new package was added.

Comment (anuragnegi000) [reply package-lock.json:1]: i haven't changed any of package.json files, but still i have commited the latest code present in upstream (talawa-admin) develop branch

**Comment by anuragnegi000** [comment]
@noman2002 please look into this PR

**Comment by anuragnegi000** [comment]
@palisadoes please look into this PR

**Comment by palisadoes** [comment]
Please add a video showing that this works with varying screen and window sizes plus screen orientations. We've had issues in the past where these factors were not tested.

**Comment by anuragnegi000** [comment]
> Please add a video showing that this works with varying screen and window sizes plus screen orientations. We've had issues in the past where these factors were not tested.

Okay

**Comment by anuragnegi000** [comment]
still working on this:-
this is effecting other screens so will create a new PR soon

**Pull request closed without merging**